docs/admin/resource-quota.md

```diff
@@ -40,15 +40,20 @@ Resource quotas are a tool for administrators to address this concern. Resource
 work like this:
 - Different teams work in different namespaces.  Currently this is voluntary, but
   support for making this mandatory via ACLs is planned.
-- Users put [compute resource limits](../user-guide/compute-resources.md) on their pods.
 - The administrator creates a Resource Quota for each namespace.
+- Users put [compute resource *requests*](../user-guide/compute-resources.md) on their pods.
+  The sum of all resource *requests* of pods belonging to the same namespace should not surpass
+  the Resource Quota for the namespace. Note that we used to verify Resource Quota by checking
+  resource *limits* of the pods, but we altered to *requests* in
```

Comment:
I think I would rephrase this to be more like the following.

Users put compute resource requests on their pods.  The sum of all resource requests across all pods in the same namespace must not exceed any hard resource limit in any Resource Quota document for the namespace.  Note that we used to verify Resource Quota by taking the sum of resource limits of the pods, but this was altered to use resource requests.  Backwards compatibility for those pods previously created is preserved because pods that only specify a resource limit have their resource requests defaulted to match their defined limits.  The user is only charged for the resources they request in the Resource Quota versus their limits because the request is the minimum amount of resource guaranteed by the cluster during scheduling.  For more information on over commit, see http://releases.k8s.io/HEAD/docs/user-guide/compute-resources.md